src/components/PageLayout/PasswordResetRequest/PasswordResetRequest.tsx

```diff
@@ -40,19 +39,25 @@ export const PasswordResetRequestForm: FC<LoginFormWrapperProps> = ({closeOverla
 
   return (
     <form onSubmit={handleSubmit(onSubmit)}>
-      <FormInput
-        control={control}
-        name="email"
-        label="Email"
-        rules={{
-          ...requiredRule,
-          pattern: {
-            value: /^[\w%+.-]+@[\d.a-z-]+\.[a-z]{2,}$/iu,
-            message: '* Vložte správnu emailovú adresu.',
-          },
-        }}
-      />
-      <Button type="submit">Resetovať heslo</Button>
+      <Stack gap={2}>
+        <FormInput
+          control={control}
+          name="email"
+          label="Email"
+          rules={{
+            ...requiredRule,
+            pattern: {
+              value: /^[\w%+.-]+@[\d.a-z-]+\.[a-z]{2,}$/iu,
+              message: '* Vložte správnu emailovú adresu.',
+            },
+          }}
+        />
+        <Stack alignItems="center" mt={2}>
```

Comment:
ked si nastavil ze `gap` bude 2, a potom `mt` 2 tak sa to scitava.. mozes pouzit len jedno z toho a nastavit to na 4 ;-)